composer.json

```diff
@@ -28,20 +28,21 @@
 		"mediawiki/data-values": "*"
 	},
 	"autoload": {
-		"files" : [
-			"DataValues.php"
-		],
 		"classmap": [
-			"src/interfaces",
-			"tests/phpunit/DataValueTest.php"
```

Comment:
:+1: it's a nice chance with a major release for any other package that needed this class to change that finally